### ISSUE TITLE
Ballerina test results has a `null` output

### ISSUE THREAD

**Issue by ldclakmal** [ballerina-platform/ballerina-lang#17347]
**Description:**
There is a `null` output when printing Ballerina test results.

Run https://github.com/ballerina-platform/ballerina-lang/blob/master/examples/secured-service-with-basic-auth/secured_service_with_basic_auth.bal file as follows:
`ballerina run -c sample-users.toml secured_service_with_basic_auth.bal`

Test https://github.com/ballerina-platform/ballerina-lang/blob/master/examples/secured-service-with-basic-auth/tests/secured_service_with_basic_auth_test.bal file as follows:
`ballerina test secured_service_with_basic_auth_test.bal`

The output is as follows:
```
Compiling source
    secured_service_with_basic_auth_test.bal

Running tests
    null
        3 passing
        0 failing
        0 skipped
```

**Comment by hemikak** [comment]
The examples need to converted to projects in order to run tests.

**Comment by Ibaqu** [comment]
Issue is resolved with #20874
